.github/workflows/spelling.yaml

```diff
@@ -0,0 +1,34 @@
+name: Check Spelling
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Why filter to these particular branches?

Reply:
No good reason. Changed to `on: pull request`